packages/terra-status/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Good point. I'd like to do that work as a separate PR. I can reach out to @aganapuram and she if she's interested in taking that task on. I'll get a issue logged for the work as well.